lms/static/scripts/frontend_apps/components/ChapterList.tsx

```diff
@@ -39,17 +39,20 @@ export default function ChapterList({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  const columns = [
-    {
-      label: 'Title',
-      field: 'title',
-    },
-    {
-      label: 'Location',
-      field: 'page',
-      classes: 'w-32',
-    },
-  ];
+  const columns = useMemo(
+    () => [
+      {
+        label: 'Title',
+        field: 'title',
+      },
+      {
+        label: 'Location',
+        field: 'page',
+        classes: 'w-32',
+      },
+    ],
+    []
+  );
```

Comment:
This value is static so it could just be extracted to a top-level variable in the module. Using a `useMemo` does have the slight advantage of being less refactoring if any dynamic elements were added in future. 🤷‍♂️